src/rust/engine/options/src/id.rs

```diff
@@ -27,7 +27,7 @@ pub(crate) fn is_valid_scope_name(name: &str) -> bool {
 impl Scope {
     pub fn named(name: &str) -> Scope {
         match name {
-            "GLOBAL" => Scope::Global,
+            "" | "GLOBAL" => Scope::Global,
```

Comment:
This is to avoid a common pitfall (which I fell into) where on the Python side the global scope is usually represented by an empty string.

In fact, I think we may want to get rid of this `GLOBAL` representation, but that is for a future change.